schema/rdl_schema_0.1.json

```diff
@@ -173,6 +212,10 @@
                   "minItems": 1,
                   "uniqueItems": true
                 },
+                "process_type": {
+                  "$ref": "#/$defs/common_process_type",
+                  "description": "Type of hazard process modelled"
+                },
```

Comment:
This should be modelled as a string field that references `hazard_process_type.csv` directly. For an example, see https://github.com/GFDRR/rdl-standard/blob/d7391d0dcb4ea780857c353044c97eb54d42db50/schema/rdl_schema_0.1.json#L1618-L1653

Once `common_process_type` is no longer referenced by any fields, it can be removed from `$defs`.

Reply:
Updated